packages/dbal-toggle/composer.json

```diff
@@ -0,0 +1,65 @@
+{
+    "name": "pheature/dbal-toggle",
```

Comment:
Refer to file `composer.json` https://github.com/pheature-flags/pheature-flags/pull/98/files#r615274845